plugwise/smile.py

```diff
@@ -178,20 +179,19 @@ def _device_data_climate(self, details, device_data):
 
         return device_data
 
-    def _get_device_data(self, dev_id):
+    def _get_device_data(self, dev_id) -> dict[str, Any]:
         """Helper-function for _all_device_data() and async_update().
         Provide device-data, based on Location ID (= dev_id), from APPLIANCES.
         """
-        devices = self._devices
-        details = devices.get(dev_id)
+        details: dict[str, Any] = self._devices.get(dev_id)
```

Comment:
The devices dictionary should be types, do local typing isn't needed here.

Reply:
Agree.